resources/qml/Dialogs/AboutDialog.qml

```diff
@@ -181,6 +197,31 @@ UM.Dialog
         }
     }
 
+    AboutDialogVersionsList{
+        id: projectBuildInfoList
+
```

Comment:
Redundant line
```suggestion
```

Reply:
Scope of AboutDialogVersionList is made local to the file using this